modules/tensor_mechanics/doc/content/source/postprocessors/MaterialTensorAverage.md

```diff
@@ -0,0 +1,34 @@
+# Material Tensor Average
+
+!syntax description /Postprocessors/MaterialTensorAverage
+
+## Description
+
+The `MaterialTensorAverage` postprocessor computes the volume average of the
+Rank-2 tensor component specified by the user.
```

Comment:
Is this only limited to rank-2? I think it would work for rank-4 as well, correct?

Reply:
This is limited to rank tow as it inherits from RankTwoTensor as it inherits from [MaterialTensorIntegral.C](https://github.com/idaholab/moose/blob/next/modules/tensor_mechanics/src/postprocessors/MaterialTensorIntegral.C)
I guess it could be expanded but I don't know anyone that is going to use it.